chainercv/utils/iterator/apply_semantic_segmentation_link.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+from chainercv.utils.iterator.split_iterator import split_iterator
+
+
+def apply_semantic_segmentation_link(target, iterator, hook=None):
+    """Apply a semantic segmentation link to an iterator
+
+    This function applies a semantic segmentation link to an iterator.
+    It stacks the outputs of the semantic segmentation link
+    into :obj:`pred_labels`.
+    This function also stacks the values returned by the iterator.
+    These values can be used for evaluation.
+
+    Args:
+        target (chainer.Link): A semantic segmentation link. This link must
+            have :meth:`predict` method which take a list of images and returns
+            :obj:`labels`.
+        iterator (chainer.Iterator): An iterator. Each sample should have
+            an image as its first element. This image is passed to
+            :obj:`target`. The rests are stacked into :obj:`gt_values`.
```

Comment:
The rests are stacked against :obj:`gt_values`.

Note: It seems that there is no usage of `stacked into`.